src/components/modules/TagPostsOverlapTitle/TitleRow.tsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import { StyleSheet } from 'react-native';
+
+import { ModuleTitle } from '@/components/modules';
+import { Leaderboard, MobileMrec } from '@/components/ads';
+
+import { Row, View } from '@/components/primitives';
+
+import { SECTION_SPACERS, SECTION_SPACING_VARIANTS } from '@/constants';
+
+type Props = {
+  patternColor: string;
+  title: string;
+  link: string;
+  isDesktop: boolean;
+  onLayout?: any;
+};
+
+class TitleRow extends PureComponent<Props> {
+
+  renderMobile = () => {
+    const showPattern = false;
+    return (
+      <>
+        <Row style={{ alignItems: 'center' }}>
+          <MobileMrec />
+        </Row>
+        <Row
+          style={{
+            marginTop: SECTION_SPACERS[SECTION_SPACING_VARIANTS.SMALL],
+            alignItems: 'center',
+            paddingHorizontal: 40
+          }}
+        >
+          {this.renderTitle(showPattern)}
+        </Row>
+      </>
+    );
+  };
+  renderDesktop = () => {
+    return (
+      <View style={styles.titleRow}>
+        <View style={{ alignItems: 'center'}}>
+          <Leaderboard />
+        </View>
+
+        <View style={styles.titleWrap}>{this.renderTitle()}</View>
+      </View>
+    );
+  };
+
+  renderTitle = (showPattern?: boolean) => {
+    const { link, title, patternColor , onLayout } = this.props;
+    // XXX should moduletitle get isDesktop?
```

Comment:
Not sure if this comment refers to showPattern or to the comment in the file. I removed the comment in the file because it's actually from TagPostsSingleColumn/TitleRow.tsx